content/fr/docs/concepts/services-networking/ingress.md

```diff
@@ -448,7 +448,7 @@ Suivez [SIG network](https://github.com/kubernetes/community/tree/master/sig-net
 Vous pouvez exposer un service de plusieurs manières sans impliquer directement la ressource Ingress :
 
 * Utilisez [Service.Type=LoadBalancer](/docs/concepts/services-networking/service/#loadbalancer)
-* Utilisez [Service.Type=NodePort](/docs/concepts/services-networking/service/#nodeport)
+* Utilisez [Service.Type=NodePort](/docs/concepts/services-networking/service/#type-nodeport)
```

Comment:
Can you modify it to use the links to the FR docs? 
* Utilisez [Service.Type=LoadBalancer](/fr/docs/concepts/services-networking/service/#loadbalancer)
* Utilisez [Service.Type=NodePort](/fr/docs/concepts/services-networking/service/#type-nodeport)

Reply:
Done